src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.StateManager.cs

```diff
@@ -190,53 +188,6 @@ public ImmutableArray<StateSet> CreateBuildOnlyProjectStateSet(Project project)
                 return stateSets.ToImmutable();
             }
 
-            public static bool OnDocumentReset(IEnumerable<StateSet> stateSets, TextDocument document)
```

Comment:
these static methods moved to be local functions at the single locations where they were called.